packages/devtools_app/lib/src/screens/vm_developer/vm_object_model.dart

```diff
@@ -208,7 +209,18 @@ class ScriptObject extends VmObject {
   SourceLocation? get _sourceLocation => null;
 
   @override
-  String? get name => null;
+  String? get name => (obj.uri ?? scriptRef?.uri)?.split('/').last;
```

Comment:
`.split('/').last` is used in several places in DevTools. Create a helper in utils.dart `fileNameFromUri(String? uri)` that returns `uri?.split('/').last`. And then file an issue to use this helper elsewhere in devtools (you can link to the TODO from a comment where you define the utility method).

Reply:
Done
https://github.com/flutter/devtools/issues/4360